site_root/input/resources/profiles/StructureDefinition-kfdrc-condition.json

```diff
@@ -0,0 +1,54 @@
+{
+    "resourceType": "StructureDefinition",
+    "id": "kfdrc-condition",
+    "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition",
+    "version": "0.1.0",
+    "name": "kfdrc-condition",
+    "title": "Kids First DRC Condition",
+    "status": "draft",
+    "fhirVersion": "4.0.0",
+    "kind": "resource",
+    "abstract": false,
+    "type": "Condition",
+    "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Condition",
+    "derivation": "constraint",
+    "differential": {
+      "element": [
+        {
+          "id": "Condition",
+          "path": "Condition"
+        },
+        {
+          "id": "Condition.extension",
+          "path": "Condition.extension"
+        },
+        {
+          "id": "Condition.extension:age-at-event",
+          "path": "Condition.extension",
+          "sliceName": "age-at-event",
+          "sliceIsConstraining": false,
+          "label": "age_at_event",
+          "short": "Patient age at determination",
+          "definition": "Age of the Patient when their condition was registered",
+          "min": 0,
+          "max": "1",
+          "type": [
+            {
+              "code": "Extension",
+              "profile": [
+                "http://fhir.kids-first.io/StructureDefinition/age-at-event"
+              ]
+            }
+          ],
+          "mustSupport": true,
```

Comment:
I think this is too strong of a constraint in a base profile. it's probably more appropriate for the "API profile". see: https://www.hl7.org/fhir/elementdefinition-definitions.html#ElementDefinition.mustSupport

Reply:
But it's the only differentiator between kfdrc condition and Condition

Reply:
Its the only diff, what about the `age-at-event` extension on `kfdrc-condition`

Reply:
Right, the profile is adding a slice to "extension" and that's enough by itself.

Reply:
I mean that age-at-event is the only difference. if you don't force it, then what's the point?

Reply:
it's not about forcing. it's about validation. when a data instance claims that it validates according to a profile, and it has an extension, the server needs to retrieve the StructureDefinition to check of the extension in the instance is valid according to the profile. and by valid i mean is the extension allowed, are the cardinalities in the data valid compared to what the profile says, any binding validation, etc.  so, it's about validation and meaning rather than structure.